code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -149,6 +149,8 @@
 /datum/pipeline/proc/merge(datum/pipeline/parent_pipeline)
 	if(parent_pipeline == src)
 		return
+	if(!parent_pipeline) //THIS RUNTIMED SO MUCH HOLY FUCK REEEEEEEE
+		return
 	air.volume += parent_pipeline.air.volume
```

Comment:
try `air.volume += parent_pipeline?.air.volume` instead. don't need the extra 2 lines probably